.github/workflows/update-config.yml

```
@@ -0,0 +1,108 @@
+name: Pull Request Action
+permissions: write-all
+on:
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ secrets.PAT_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      
+      - name: Set up git
+        run: |
+          git config --global user.email "UbiquiBot[dev]@github.com" 
```

```suggestion
          git config --global user.email "UbiquiBot[bot]@github.com" 
```
I think that's the correct username for the associated secret